packages/packaging/meta.yaml

```diff
@@ -4,8 +4,12 @@ package:
   top-level:
     - packaging
 source:
-  sha256: ef103e05f519cdc783ae24ea4e2e0f508a9c99b2d4969652eed6a2e1ea5bd522
-  url: https://files.pythonhosted.org/packages/05/8e/8de486cbd03baba4deef4142bd643a3e7bbe954a784dc1bb17142572d127/packaging-21.3-py3-none-any.whl
+  sha256: dd47c42927d89ab911e606518907cc2d3a1f38bbd026385970643f9c5b8ecfeb
+  url: https://files.pythonhosted.org/packages/df/9e/d1a7217f69310c1db8fdf8ab396229f55a699ce34a203691794c5d1cad0c/packaging-21.3.tar.gz
+
+  patches:
+    - patches/allow-cpp310-none-any-tag.patch
```

Comment:
Packaging 0.22.0 is released now, so probably we can go ahead and update.

Reply:
For now, we are still blocked by https://github.com/pypa/packaging/issues/638 due to https://github.com/pyodide/micropip/issues/36 so we would need to wait for 0.22.1.

I'll update the comment in the patch.